### ISSUE TITLE
cache save path

### ISSUE THREAD

**Issue by Edotor** [fast4x/RiMusic#1429]
### Description

If it is possible I'd like to have this mod, to select the path for storing the cache of offline music and thumb image. This because I have limited memory in android car radio, and I'd like to set pendrive memory. Thanks a lot

### Suggested Solution

For better use is good to choose path

**Comment by fast4x** [comment]
From Android 10 there's a several limitations for this, I'm not sure if it is possible.

**Comment by Edotor** [comment]
Yes, I know, but please try to do it because if you can make it works, it will be an excellent thing! And the app will be more user friendly with all types of devices. Thanks in advance

**Comment by Edotor** [comment]
Are there some news Rino? Spero tu riesca a farlo

**Comment by fast4x** [comment]
I'll try

**Comment by Edotor** [comment]
I find this... I danno if it can help you

https://developer.android.com/about/versions/11/privacy/storage?hl=it
